package.json

```diff
@@ -173,81 +178,99 @@
     "electron": "./node_modules/.bin/electron"
   },
   "devDependencies": {
+    "@babel/core": "7.7.2",
+    "@babel/plugin-proposal-class-properties": "7.7.0",
+    "@babel/plugin-proposal-decorators": "7.7.0",
+    "@babel/plugin-proposal-do-expressions": "7.6.0",
+    "@babel/plugin-proposal-export-default-from": "7.5.2",
+    "@babel/plugin-proposal-export-namespace-from": "7.5.2",
+    "@babel/plugin-proposal-function-bind": "7.2.0",
+    "@babel/plugin-proposal-function-sent": "7.7.0",
+    "@babel/plugin-proposal-json-strings": "7.2.0",
+    "@babel/plugin-proposal-logical-assignment-operators": "7.2.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "7.4.4",
+    "@babel/plugin-proposal-numeric-separator": "7.2.0",
+    "@babel/plugin-proposal-optional-chaining": "7.6.0",
+    "@babel/plugin-proposal-pipeline-operator": "7.5.0",
+    "@babel/plugin-proposal-throw-expressions": "7.2.0",
+    "@babel/plugin-syntax-dynamic-import": "7.2.0",
+    "@babel/plugin-syntax-import-meta": "7.2.0",
+    "@babel/plugin-transform-classes": "7.7.0",
+    "@babel/plugin-transform-react-constant-elements": "7.6.3",
+    "@babel/plugin-transform-react-inline-elements": "7.2.0",
+    "@babel/polyfill": "7.7.0",
+    "@babel/preset-env": "7.7.1",
+    "@babel/preset-react": "7.7.0",
+    "@babel/register": "7.7.0",
+    "@babel/runtime-corejs2": "7.7.2",
+    "@jest-runner/electron": "2.0.2",
     "@ronomon/direct-io": "2.4.3",
     "aws-sdk": "2.472.0",
-    "babel-core": "6.26.3",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
What do we need to do to get rid of this bridge in future?

Reply:
Output from `yarn why babel-core` reveals `Hoisted from "@jest-runner#electron#@jest-runner#rpc#jscodeshift#babel-core"`, as the only dependency still pointing at `babel-core` instead of `@babel/core`, so once that dependency is updated to point to the new dependency path, we can drop this. (additionally `babili-webpack-plugin` was pointing at it, but is an unused dependency so I'll remove it)